sklearn/multioutput.py

```diff
@@ -1166,6 +1202,12 @@ class RegressorChain(MetaEstimatorMixin, RegressorMixin, _BaseChain):
 
         .. versionadded:: 1.2
 
+    base_estimator : estimator, default="deprected"
+        Use `estimator` instead.
+        .. deprected:: 1.5
+            `base_estimator` is deprected and will be removed in 1.7.
```

Comment:
```suggestion
    base_estimator : estimator, default=None
        Use `estimator` instead.
        .. deprecated:: 1.6
            `base_estimator` is deprecated and will be removed in 1.8.
```

Typos and versions, but also note that we comunitate `None` to the users as a default. `"deprecated"` is for internal usage.
This also applies above in `ClassifierChain`.